src/Raytracer/Entity/Scene.hpp

```diff
@@ -12,23 +12,24 @@
 #include <memory>
 #include "IEntity.hpp"
 #include "Image.hpp"
+#include "EventManager.hpp"
 
 namespace Raytracer {
     class Scene {
     public:
         Scene();
 
-        ~Scene();
+        ~Scene() = default;
```

Comment:
use it to destroy all entities with
`destroyEntity(Raytracer::IEntity *entity);`